Terminal.Gui/Core/PosDim.cs

```diff
@@ -361,7 +361,7 @@ internal virtual int Anchor (int width)
 			return 0;
 		}
 
-		class DimFactor : Dim {
+		internal class DimFactor : Dim {
```

Comment:
This is the first class where access to these internals have appeared to be needed. This makes me very nervous as @migueldeicaza was very careful in the design. 

In all of my work on `Terminal.Gui` I've been able to find out ways to make things like what you are trying here work without accessing these internals.

What is it about ComboBox that requires this change?